src/data_ideas.md

```diff
@@ -0,0 +1,4 @@
+- check if a token exists
```

Comment:
I would either put these ideas in the issue or create a wiki.

Reply:
We tossed these data idea type files